package.json

```diff
@@ -23,6 +23,8 @@
     "@trpc/next": "^10.15.0",
     "@trpc/react-query": "^10.8.1",
     "@trpc/server": "^10.15.0",
+    "daisyui": "^2.51.3",
+    "esbuild": "^0.17.11",
```

Comment:
Why do we need these libraries?

Reply:
these deps were from https://github.com/CPSC319-2022/Continuus/pull/33 strange how they got in here 🤔

Reply:
oh lol I worked off your branch that's why, I'll remove it